### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by MuhammadSanaullah123** [community/community#53923] [GitHub Education]
### Hi everyone,

Hello. I tried applying for the Student Developer Pack but when they ask me documents of proof of affiliation with the institution and when I opened the camera, back camera does not show even when I choose the back camera option. Only front camera is being opened and I cannot take pictures with the front camera. I tried but it was blurry. I tried it on my phone by the way.

**Comment by Saran-pariyar** [comment]
I'm facing same problem, have you got the solution bro?
The back camera is not opening and no matter what I do, the front camera only takes blurry picture

**Reply by MuhammadSanaullah123** [reply]
I couldn't find the solution so I tried it on my laptop. I took picture of my student card through the laptop camera and it was better than the mobile front camera so it got approved. I recommend trying on laptop first.
